tools/ci_build/github/linux/docker/Dockerfile.ubuntu_openvino

```diff
@@ -47,4 +48,4 @@ ARG BUILD_USER=onnxruntimedev
 WORKDIR /home/$BUILD_USER
 RUN adduser --gecos 'onnxruntime Build User' --disabled-password $BUILD_USER --uid $BUILD_UID
 RUN adduser $BUILD_USER video
```

Comment:
I think this is why the openvino build failed. It created two users! The one used for running CI build doesn't have the necessary permissions. And the one has perm wasn't used for doing anything.

Reply:
Ah ok. Well that sounds like a problem. To re-enable the java tests I think all that's necessary is to add `--build_java` to the end of the docker run command in linux-openvino-ci-pipeline.yml, inside the double quotes.